bridge/engine/AudioForwarderRewriteAndSendJob.cpp

```diff
@@ -1,19 +1,54 @@
 #include "bridge/engine/AudioForwarderRewriteAndSendJob.h"
+#include "bridge/engine/SsrcInboundContext.h"
 #include "bridge/engine/SsrcOutboundContext.h"
 #include "codec/Opus.h"
 #include "rtp/RtpHeader.h"
 #include "transport/Transport.h"
 #include "utils/OutboundSequenceNumber.h"
 
+namespace
+{
+
+inline void rewriteHeaderExtensions(rtp::RtpHeader* rtpHeader,
+    const bridge::SsrcInboundContext& senderInboundContext,
+    const bridge::SsrcOutboundContext& receiverOutboundContext)
+{
+    assert(rtpHeader);
+
+    const auto headerExtensions = rtpHeader->getExtensionHeader();
+    if (!headerExtensions)
+    {
+        return;
+    }
+
+    for (auto& rtpHeaderExtension : headerExtensions->extensions())
+    {
+        if (senderInboundContext._rtpMap._audioLevelExtId.isSet() &&
+            rtpHeaderExtension.getId() == senderInboundContext._rtpMap._audioLevelExtId.get())
+        {
+            rtpHeaderExtension.setId(receiverOutboundContext._rtpMap._audioLevelExtId.get());
```

Comment:
Now we are not checking that the outboundcontext really has a rtp hdr ext id set.
I wonder what happens with ext hdrs SMB do not care about. They will pass through unaltered and it works as long as all the clients have the same idea about the mapping. We have clients that offer other ext.

Reply:
True, this is the same behavior as before though. A server receiving an offer should filter out all unsupported extensions in the answer. A server sending an offer will only send the supported ones.